packages/react-scripts/scripts/start.js

```diff
@@ -85,6 +85,12 @@ choosePort(HOST, DEFAULT_PORT)
       }
       console.log(chalk.cyan('Starting the development server...\n'));
       openBrowser(urls.localUrlForBrowser);
+      if (process.platform === 'win32') {
```

Comment:
I think this would still block the first compilation, no? It would be nice to check this (e.g. by artificially putting an infinite loop into the launching code). Perhaps `setTimeout` would help?